cmake/configs/wiredtiger_config.h.in

```diff
@@ -143,6 +143,9 @@
 /* Define to 1 if you have the <memory.h> header file. */
 #cmakedefine HAVE_MEMORY_H 1
 
+/* Define to 0 for have RCpc. */
```

Comment:
```suggestion
/* Define to 1 for inline assembly that uses RCpc. */
```

Reply:
This is a bit weird because these seem to be all "if you have". So I'll update to:
"Define to 1 if your hardware supports ARM RCpc.

Reply:
795fc737e